pkg/server/apis/types.go

```diff
@@ -47,9 +47,9 @@ type WorkloadRunBody struct {
 
 // WorkloadMeta store workload metadata for dashboard restful API server
 type WorkloadMeta struct {
-	Name       string            `json:"name"`
-	Parameters []types.Parameter `json:"parameters,omitempty"`
-	AppliesTo  []string          `json:"appliesTo,omitempty"`
```

Comment:
why delete appliesTo?

Reply:
BTW, it's better to have a unit test for it. Now, even we delete some field(which should be a break change) that the CI won't fail.

Reply:
Workload should not have `AppliesTo` field. And I have confirmed all callers of this struct and it's only called by `pkg/server/workloadHandler.go`.